ext/bg/js/settings.js

```diff
@@ -237,6 +243,18 @@ function dictionaryGroupsSort() {
     dictGroups.append(dictGroupChildren);
 }
 
+function dictionarySetMain(e) {
+    const mainDictionary = $(e.target).closest('.dict-group');
+    const mainDictionaryTitle = mainDictionary.data('title');
+
+    $('.dict-group').each((index, element) => {
+        const dictionary = $(element);
+        if (dictionary.data('title') !== mainDictionaryTitle) {
```

Comment:
I don't think that this should be necessary if all of the radio buttons are in the same group (set by `name` attribute). Setting one should unset the others.